bindings/java/src/main/java/org/openucx/jucx/UcxUtils.java

```diff
@@ -27,10 +27,6 @@
     /**
      * Returns view of underlying memory region as a ByteBuffer.
      * @param address - address of start of memory region
-     * @param length
```

Comment:
why removed this?

Reply:
Just to remove Warnings during javadoc builds.